core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -202,15 +203,18 @@ var TopicEditorPage = function() {
     await saveSubtopicButton.click();
   };
 
-  this.changeSubtopicPageContents = async function(richTextInstructions) {
-    var subtopicPageEditor = await forms.RichTextEditor(
-      pageEditor);
-    await subtopicPageEditor.clear();
-    await richTextInstructions(subtopicPageEditor);
-  };
-
-  this.editSubtopicWithIndex = async function(index) {
-    await editSubtopicButtons.get(index).click();
+  this.changeSubtopicPageContents = async function(content) {
```

Comment:
From your screenshots it looks like the subtopic page still supports rich text. Why are we limiting to plaintext here?

Reply:
The richText syntax wasn't working for some reason, so I looked at other schema-based-editors, and they were following the same pattern as what i did here.

Reply:
Also, if you want this changes, I can do that in the next PR that i have [here](https://github.com/oppia/oppia/pull/9672)

Reply:
@Showtim3 yeah, that'd be great. Testing with the RTE isn't super critical since it's tested extensively elsewhere in the e2e tests, but it's nice to make sure it works here too